flink-libraries/flink-table/src/test/scala/org/apache/flink/table/descriptors/SchemaValidatorTest.scala

```diff
@@ -69,14 +69,6 @@ class SchemaValidatorTest {
     assertEquals(
       expectedMapping,
       SchemaValidator.deriveFieldMapping(props, Optional.of(inputSchema.toRowType)))
-
```

Comment:
Why is this part of the test removed?

Reply:
I moved the test to `org.apache.flink.table.factories.TableFormatFactoryBaseTest#testSchemaDerivation`.